articles/api/management/v2/tokens.md

```diff
@@ -1,86 +1,278 @@
 ---
 description: Details generating and using an Auth0 Management APIv2 token.
 section: apis
+toc: true
 ---
 
 # The Auth0 Management APIv2 Token
 
 ## Overview
 
-The Auth0 Management APIv2 token is required to call v2 of the Auth0 Management API.  This token is used by a specific tenant in Auth0 to call Auth0 Management APIv2 to access or update records for that tenant.  This Management APIv2 token is a JWT, and contains various scopes, such as `read:users` or `update:clients`, and is signed with a client API key and secret for the entire tenant.
+In order to call the endpoints of [Auth0 Management API v2](/api/management/v2), you need a token, what we refer to as Auth0 Management APIv2 Token. This token is a [JWT](/jwt), contains specific granted permissions (known as __scopes__), and is signed with a client API key and secret for the entire tenant.
 
-## How to get a Management APIv2 Token
+There are two ways to get a Management APIv2 Token: [get one manually using the Dashboard](#get-a-token-manually), or [automate the process](#automate-the-process) (build a simple command line tool that generates tokens). In this article we will see how you can do either.
 
-**Create a Non Interactive Client**
+## Get a token manually
 
-A Non Interactive Client is a client that interacts with an API where there is no user involved. It's a machine to machine interaction. This must be used instead of a Single Page or Native apps because those cannot meet the necessary security requirements for executing this type of flow.
+<div class="alert alert-info">
+  <strong>Heads up!</strong> Before you decide to follow the manual process, you should know that the Management APIv2 token has a validity of <strong>24 hours</strong>. After that the token will expire and you will have to get a new one. If this doesn't work for you, you can <a href="#automate-the-process">automate the process</a>.
```

Comment:
I would go with the Warning background instead (yellow one) since this is a breaking change.